src/selenium.ts

```diff
@@ -0,0 +1,163 @@
+import {
+  Builder,
+  By,
+  FileDetector,
+  IWebElement,
+  WebDriver,
+  until,
+} from 'selenium-webdriver';
+import chrome from 'selenium-webdriver/chrome';
+import { Origin } from 'selenium-webdriver/lib/input';
+
+import { Context, STARTUP_STORAGE_KEY } from './sandbox';
+
+const WAIT_TIMEOUT = 1000 * 10;
+
+export type Config = {
```

Comment:
I feel like for exported things interfaces are better? Since they are more powerful and we don't know how they will be used...w/d/t?

Reply:
I think you are right. I don't think you can do declaration merging with types, but you can with interfaces